sources/projects/base64-encode-decode/requirements.md

```diff
@@ -0,0 +1,21 @@
+Write a program that accepts two parameters: the mode (`encode` or `decode`) and
+some text.
+
+- If the mode is `decode`, it should print the decoded Base64 text.
+- If the mode is `encode`, it should print the encoded text.
+- It should print an error message and exit with an error when the input is
+  invalid.
+
+```console
+$ ./base64-encode-decode.lang "encode" "hello world"
```

Comment:
I think this is the correct way to put this together. In my head, I thought maybe it might be easy to tell if a string needs to be decoded or not (so you wouldn't have to specify it here), but adding that logic as a requirement is too much.

Reply:
Detecting that would be difficult, and it is possible that the encode string could look like something that is valid base64. For example, "kite" is valid base64.